technical guides/Walls.md

```diff
@@ -0,0 +1,258 @@
+| T1 Cons | Useful on front line, can build popups | Slow, vulnerable, expensive |
+| Minelayers | Can also build mines, cheap, fast, has local jamming | Cannot support each other, restricted to T1 vehicle lab |
+| T2 Cons | Can build highest tier T2 defenses that synergize with fortress walls | Expensive, vulnerable, slow |
+| Combat Engineers | Can build tall T2 defenses that syndergize with fortress walls, fast, good for reclaim as well, build mines | Restricted to specific T2 labs |
+
+- Minelayers are ideal for building T1 walls, and combat engineers are ideal for building T2 walls. However, these are specific to certain factories. Arm bot players will see neither of these options.
+- In the early game, the commander can quickly build a bunker for artillery or plasma to fight vs rocket bots
+## Walls as Protection / Cover
+Walls can block damage from some units against some units. Technically, whether a weapon will be blocked by a wall depends on whether the projectile's path from the weapon to the target's "aimposition" will be blocked by the wall's collision volume. In layman's terms, this means some units (usually ones lower to the ground) will be easier to protect with walls, and some units (usually ones with arcing shots, e.g. plasma bots) will be harder or impossible to defend against using walls. 
+
+![Walled Arty](https://i.imgur.com/txonxoj.gif)
+
+You can use the /debugcolvol command to view target collision. There will be a dot indicating the aimposition, which is where enemies will aim to destroy the unit. For most T1 units (including windmills, oddly enough), this is close to the ground and about level with Dragon's Teeth, so they'll be safe vs direct fire weapons behind walls. In T1 this includes all scout and raider units as well as missile trucks, rocket bots, and Centurions.
+
+![Debugcolvol](https://i.imgur.com/lMiB7cF.png)
+
+![Debugcolvol lineup](https://i.imgur.com/0SxcDTX.png)
+
+- Constructors will be safe from rocket bots and pawns/grunts when behind walls, allowing you to build popup turrets, or more walls
+- *This is especially powerful for T1 popup turrets (Dragon's Claw / Maw), which are not only disguised as walls but easily protected by them as well.*
+- ![Walled Popup](https://i.imgur.com/jvT0d2E.gif)
+  - Side note on T1 popup turrets: make sure you set the T1 popup turrets building facing towards the enemy so they extend to fire and retract much more quickly. T2 popup turrets preserve their facing.
+<spoiler>
+<details>Note on T2 popup turrets</details>
+T2 popup turrets are also protected by T1 walls, but there is a bit of unintended behavior with Pitbulls (Arm T2 popup). If there is a wall adjacent to it and between it and its target, it will not pop up to fire. If it is already popped up (e.g. from firing beforehand), it will fire on units in range. Keep this in mind if you choose to protect pitbulls with walls.
+</spoiler>

```

Comment:
This `<spoiler> tag is not appearing, at least, in the Github markdown display. What about simply
```suggestion
<details>
<summary>Note on T2 popup turrets</summary>
T2 popup turrets are also protected by T1 walls, but there is a bit of unintended behavior with Pitbulls (Arm T2 popup). If there is a wall adjacent to it and between it and its target, it will not pop up to fire. If it is already popped up (e.g. from firing beforehand), it will fire on units in range. Keep this in mind if you choose to protect pitbulls with walls.
</details>
```